src/main/java/org/sagebionetworks/web/client/jsinterop/QueryWrapperPlotNavProps.java

```diff
@@ -66,7 +66,7 @@ public static QueryWrapperPlotNavProps create(
     props.shouldDeepLink = false;
     props.name = "Items";
     props.downloadCartPageUrl = "#!DownloadCart:0";
-    props.showLastUpdatedOn = true;
+    props.showLastUpdatedOn = false;
```

Comment:
Hide the `lastUpdatedOn` in the QueryWrapperPlotNav because the new component now shows it. Behavior unchanged for StandaloneQueryWrapper, which is not shown on an entity page, so the information is still visible.